src/types/onyx/TransactionViolation.ts

```diff
@@ -0,0 +1,34 @@
+/**
+ * @module TransactionViolation
+ * @description Transaction Violation
+ */
+
```

Comment:
```suggestion
```
Is this part of any of our guidelines or can we get rid of this? I think it's just noise and doesn't add anything

Reply:
Ah sorry this is boilerplate from my template file -- will remove